dolphin/resource_manager/storages.py

```diff
@@ -52,3 +60,48 @@ def get_all(req):
         if len(storage_all) == 0:
             break
     return build_storages(storage_all)
+
+
+def register(self, req, body):
+    """
+    This is create function for registering the new storage device
+    :param req:
+    :param body: "It contains the all input parameters"
+    :return:
+    """
+    # Check if body is valid
+    if not wsgi.Controller.is_valid_body(body, 'storages'):
+        msg = "Storage entity not found in request body"
+        raise exc.HTTPUnprocessableEntity(explanation=msg)
+
+    required_parameters = ('name', 'manufacturer', 'model', 'username', 'password')
+
```

Comment:
required_parameters are hostname or hostip, manufacturer, model, username, password

Reply:
as per API spec, name is showing required paramaters. plz confirm, it should be name or hostname or hotip?